README.md

```diff
@@ -22,6 +21,12 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-47-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+## Distribubtion
+
+[![AUR package](https://repology.org/badge/version-for-repo/aur/datalad.svg)](https://repology.org/project/datalad/versions)
+[![Gentoo Science package](https://repology.org/badge/version-for-repo/gentoo_ovl_science/datalad.svg)](https://repology.org/project/datalad/versions)
+[![PyPI package](https://repology.org/badge/version-for-repo/pypi/datalad.svg)](https://repology.org/project/datalad/versions)
```

Comment:
oh lovely! I do love all the "automatically updating and free for us"  badges as you can see from all the ones on https://github.com/datalad/datalad-extensions which reminds me that we can get one also for conda-forge. I also duplicated Github release here since it does provide some artifacts and makes it easier to compare without running eyes up to compare to the most recent on github.

```suggestion
[![AUR package](https://repology.org/badge/version-for-repo/aur/datalad.svg)](https://repology.org/project/datalad/versions)
[![Conda-forge](https://anaconda.org/conda-forge/datalad/badges/version.svg)](https://anaconda.org/conda-forge/datalad)
[![GitHub release](https://img.shields.io/github/release/datalad/datalad.svg)](https://GitHub.com/datalad/datalad/releases/)
[![Gentoo Science package](https://repology.org/badge/version-for-repo/gentoo_ovl_science/datalad.svg)](https://repology.org/project/datalad/versions)
[![PyPI package](https://repology.org/badge/version-for-repo/pypi/datalad.svg)](https://repology.org/project/datalad/versions)
```